ha/core/config/config_manager.py

```diff
@@ -39,28 +38,35 @@ class ConfigManager:
     _conf = []
     _cluster_confstore = None
 
-    # TODO: create separate function for log and conf file
     @staticmethod
-    def init(log_name):
+    def init(log_name, log_path=None, level="INFO",
+            backup_count=5, file_size_in_mb=10, syslog_server=None,
+            syslog_port=None, console_output=False, config_file=None):
         """
         Initialize ha conf and log
         Args:
             log_name (str): service_name for log init.
         """
-        if len(ConfigManager._conf) == 0:
-            Conf.init()
-        ConfigManager._safe_load(const.HA_GLOBAL_INDEX, f"yaml://{const.HA_CONFIG_FILE}")
-        if log_name is not None:
-            ConfigManager._init_log(log_name)
+        ConfigManager._conf_init(config_file)
+        if log_name:
+            if not log_path:
+                log_path = Conf.get(const.HA_GLOBAL_INDEX, f"LOG{_DELIM}path")
+            level = Conf.get(const.HA_GLOBAL_INDEX, f"LOG{_DELIM}level")
+            Log.init(service_name=log_name, log_path=log_path, level=level,
```

Comment:
We are not populating syslog_server, syslog_port, console_ouput parameters while calling ConfigManager.init 
Are defaults picked up by Log and do they point to right values ?

Reply:
yes default are none copied from log utility